website/www/site/content/en/case-studies/schrodinger.md

```diff
@@ -0,0 +1,19 @@
+---
+title:  "Schrodinger"
+icon: /images/logos/powered-by/schrodinger.png
```

Comment:
This looks right, but it looks like the logo ended up in the wrong directory (`sdks/python` instead of https://github.com/apache/beam/tree/master/website/www/site/static/images/logos/powered-by)